analysis_module/views/analysis_module.py

```diff
@@ -12,53 +11,13 @@
     EntriesSerializer,
     NgramsRequest,
 )
-from core_server.settings import IS_MOCKSERVER, SUMMARIZATION_V2_ECS_ENDPOINT
-
+from core_server.settings import IS_MOCKSERVER, USE_NEW_SUMMARIZATION
 from core.models import NLPRequest
-from analysis_module.utils import spin_ecs_container, send_ecs_http_request
-from analysis_module.mockserver import (
-    topicmodeling_mock_model,
-    ngrams_mock_model,
-    summarization_mock_model,
-    geolocation_mock_model,
-)
+from analysis_module.utils import spin_ecs_container
+from analysis_module.mockserver import process_mock_request
 
 RequestType = Literal["topicmodel", "ngrams", "summarization", "geolocation"]
```

Comment:
do we need to include the text-extraction type here?

Reply:
Not really. Actually the name is confusing here, the request types here refer to those for which ECS is to be spinned off. I renamed the type name.

Reply:
got it.